src/scripts/i18n/i18n_extraction.py

```
@@ -0,0 +1,358 @@
+                        en_json[k] = v
+            if 'files' in data:
+                for file_obj in data['files']:
+                    content = file_obj['content']
+                    content = content.replace('<START_OF_FILE>', '').replace('<END_OF_FILE>', '')
+                    if not any(f['fileName'] == file_obj['fileName'] for f in output_files):
+                        output_files.append({
+                            'fileName': file_obj['fileName'],
+                            'content': content
+                        })
+
+def process_failed_batches(en_json, output_files, failed_batches_path, type):
```

Ruff (0.11.9)

`process_failed_batches` is too complex (12 > 10)

(C901)